gptme/tools/shell.py

```diff
@@ -250,7 +250,7 @@ def execute_shell(
 
     print_preview(cmd, "bash")
     if not confirm("Run command?"):
-        yield Message("system", "Command not run")
+        yield Message("system", "User cancelled command.")
```

Comment:
I think this is more correct:

```suggestion
        yield Message("system", "User chose not to run command")
```